akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -388,35 +378,24 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
   }
 
   private def runInboundStreams(): Unit = {
-    runInboundControlStream()
-    runInboundOrdinaryMessagesStream()
+    val compression = createInboundCompressionTable()
+
+    runInboundControlStream(compression)
+    runInboundOrdinaryMessagesStream(compression)
```

Comment:
if we start without versioning and identifiers we should only use compression for the OrdinaryMessagesStream, i.e. one table